.vscode/launch.json

```diff
@@ -4,31 +4,34 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
-        
+
         {
             "name": "Python: Current File (Integrated Terminal)",
             "type": "python",
             "request": "launch",
             "program": "${file}",
-            "console": "integratedTerminal"
+            "console": "integratedTerminal",
         },
         {
             "name": "Python: Remote Attach",
             "type": "python",
             "request": "attach",
             "port": 5678,
             "host": "localhost",
-            "pathMappings": [{
-                "localRoot": "${workspaceFolder}",
-                "remoteRoot": "."
-            }]
+            "pathMappings": [
+
```

Comment:
blank line

Reply:
Blank line is actually required in json files to allow trailing commas in arrays and to help differentiate between two separate changes.

So for example, If I have a setting for `thing1` and I add a setting for `thing2`, if we don't have trailing commas and blank spaces the commit made for `thing2` will also have changes for `thing1`.

All of the json formatting issues can probably be solved by using a formatter. I don't know any good json formatter that also supports configuration and comments.